### DISCUSSION TITLE
fibo-fnd-utl-av:MaturityLevel and explanatory notes

### DISCUSSION THREAD

**Discussion by trypuz** [edmcouncil/fibo#1373] [Foundations (FND)]
Explanatory notes for https://spec.edmcouncil.org/fibo/ontology/FND/Utilities/AnnotationVocabulary/MaturityLevel and its instances refer to FIBO. I suggest we remove "FIBO", from the notes, so that other ontologies could use our MaturityLevel too (BTW, we use it for [AUTO](https://spec.edmcouncil.org/auto/ontology)).

**Comment by rivettp** [comment]
The Maturity levels were designed specifically for the FIBO process, which is documented - so people can see the maturity levels in context. "Informative" in particular is very specific to FIBO's evolution from a legacy UML model, and in fact there are only a handful of Informative ontologies remaining (in FND/Transactions). So I would not expect, or advise, other ontologies to reuse that from FIBO.
And for FIBO ontologies, whether they are reused elsewhere or not, that process will remain a FIBO one.

If it's a general goal to remove use of "FIBO" then it will impact a lot more than AnnotationVocabulary. For example fibo-fnd-utl-alx:StatisticalMeasure which seems to be used by AUTO. 

However I'm not keen on doing this. I don't think it's wrong to have notes referencing "FIBO" since, even if it's reused elsewhere, it's important for people to understand that the element's scope was designed for FIBO (and not AUTO for example) purposes. In the case of statistics, for example, it's important for people to know we've not designed it to be suitable for use with car parts.
